cypress/e2e/create_tx.cy.js

```diff
@@ -77,6 +79,41 @@ describe('Queue a transaction on 1/N', () => {
     })
     cy.contains('Estimated fee').should('exist')
 
+    // Asserting the relayer option is present
+    cy.contains('Via relayer').find('[type="radio"]').should('be.checked')
+    cy.contains('Sponsored by Gnosis Chain')
+    cy.get('span').contains('Estimated fee').next().should('have.css', 'text-decoration-line', 'line-through')
+    cy.contains('Via relayer').contains(/[0-5] of 5/)
```

Comment:
Here i think we should assert the exact string "5 of 5" instead of a regex

Reply:
Ok, I'll make it 5 of 5 strict